draft-irtf-cfrg-frost.md

```diff
@@ -1394,7 +1394,7 @@ The procedure for splitting a secret into shares is as follows.
   - coefficients, a vector of MIN_PARTICIPANTS coefficients which uniquely determine a polynomial f.
 
   Errors:
-  - "invalid parameters", if MIN_PARTICIPANTS > MAX_PARTICIPANTS or if MIN_PARTICIPANTS is less than 2
+  - "invalid parameters", if MIN_PARTICIPANTS > MAX_PARTICIPANTS or if MIN_PARTICIPANTS is less than 1
```

Comment:
editorial: maybe `if MIN_PARTICIPANTS < 1` (because you use `>` in the first part of the sentence)